### ISSUE TITLE
SmartCity-Accessibility Server Side Network Tests #3 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#347]
Build 'SmartCity-Accessibility Server Side Network Tests' is failing!

Last 50 lines of build :
```
[...truncated 5.49 KB...]
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 15, 2017
JxMaps license valid.
Running smartcity.accessibility.services.NavigationServiceTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 19.803 sec - in smartcity.accessibility.services.NavigationServiceTest
Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 5.559 sec - in smartcity.accessibility.navigation.NavigationTest
Running smartcity.accessibility.database.ParseDatabaseTest
Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.885 sec <<< FAILURE! - in smartcity.accessibility.database.ParseDatabaseTest
testGetLocation(smartcity.accessibility.database.ParseDatabaseTest)  Time elapsed: 0.529 sec  <<< FAILURE!
java.lang.AssertionError: couldn't find the sample object
	at smartcity.accessibility.database.ParseDatabaseTest.testGetLocation(ParseDatabaseTest.java:85)


Results :

Failed tests: 
  ParseDatabaseTest.testGetLocation:85 couldn't find the sample object

Tests run: 13, Failures: 1, Errors: 0, Skipped: 1

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:32 min
[INFO] Finished at: 2017-05-15T17:16:14+00:00
[INFO] Final Memory: 33M/208M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
No changes

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Network%20Tests/3/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Network Tests' is failing!

Last 50 lines of build :
```
[...truncated 5.49 KB...]
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 15, 2017
JxMaps license valid.
Running smartcity.accessibility.services.NavigationServiceTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 20.553 sec - in smartcity.accessibility.services.NavigationServiceTest
Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 5.358 sec - in smartcity.accessibility.navigation.NavigationTest
Running smartcity.accessibility.database.ParseDatabaseTest
Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.851 sec <<< FAILURE! - in smartcity.accessibility.database.ParseDatabaseTest
testGetLocation(smartcity.accessibility.database.ParseDatabaseTest)  Time elapsed: 0.549 sec  <<< FAILURE!
java.lang.AssertionError: couldn't find the sample object
	at smartcity.accessibility.database.ParseDatabaseTest.testGetLocation(ParseDatabaseTest.java:85)


Results :

Failed tests: 
  ParseDatabaseTest.testGetLocation:85 couldn't find the sample object

Tests run: 13, Failures: 1, Errors: 0, Skipped: 1

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:27 min
[INFO] Finished at: 2017-05-15T17:21:39+00:00
[INFO] Final Memory: 33M/207M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
No changes
No changes


[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Network%20Tests/4/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
